### ISSUE TITLE
Need better error for invalid package.json of module

### ISSUE THREAD

**Issue by knownasilya** [spolu/breach_core#127]
Currently we run into [this](https://github.com/mblarsen/breach-module-manager/issues/3) when trying to install a package with an invalid `package.json`.

The error was ambiguous at first, but if you add the return it was less ambiguous, but still ambiguous.. lol. Something about a missing `}` somewhere..

**Comment by spolu** [comment]
Yes we totally need a proper error message in the core module_manager
